### PR TITLE
OSDOCS-8804: Adding the MicroShift service to a blueprint section is …

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.15+

Issue:
[OSDOCS-8804](https://issues.redhat.com/browse/OSDOCS-8804)

Link to docs preview:
[Embedding in a RHEL for image - regular install](https://70554--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree#adding-microshift-service-to-blueprint_microshift-embed-in-rpm-ostree)
[Embedding in a RHEL for Edge image - offline use](https://70554--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree-offline-use#adding-microshift-service-to-blueprint_microshift-embed-rpm-ostree-offline-use)

QE review:
- [x] QE approval is required.

### CODE REVIEW

**Comment by aravipra** [comment]
to trigger travis build reopening the PR

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 1]
```diff
@@ -9,11 +9,19 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* If your goal is a faster installation, use step 1 and create your own minimal blueprint file.
+* If your goal is a faster startup for {microshift-short}, use the generated blueprint in step 2. Using the generated blueprints is a longer installation process, but a faster start up because container references are accessed locally.
++
+[IMPORTANT]
+====
+Replace <minimal-microshift> in the following procedures with the name of the TOML you are using.
+====
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:18]: Can we remove this please and use a file name in the commands rather than `<some name>`?
The latter really annoys when copy-pasting commands for execution.

Comment (ggiguash) [reply modules/microshift-adding-service-to-blueprint.adoc:18]: We have references to this `minimal-microshift` in a few places. I'd suggest to replace it by `microshift-blueprint` to be less ambiguous.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 2]
```diff
@@ -9,11 +9,19 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* If your goal is a faster installation, use step 1 and create your own minimal blueprint file.
+* If your goal is a faster startup for {microshift-short}, use the generated blueprint in step 2. Using the generated blueprints is a longer installation process, but a faster start up because container references are accessed locally.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:13]: Faster installation and faster startup are by-products of having one or another blueprint type, but they are not goals.
I think we should rephrase this to emphasize that step 1 results a minimal microshift installation, while the step 2  includes all the RPM packages and container images. We can still mention that the former results in a faster installation, and the latter in faster startup.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 3]
```diff
@@ -9,11 +9,19 @@
...
++
+[IMPORTANT]
+====
+Replace <minimal-microshift> in the following procedures with the name of the TOML you are using.
+====
+
 .Procedure
 
-. Use the following example to create your blueprint:
+. Use the following example to create your own <minimal-microshift> blueprint file:
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:22]:
```suggestion
. Use the following example to create your own `microshift-blueprint.toml` blueprint file:
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 4]
```diff
@@ -39,21 +47,62 @@ EOF
...
+<1> References for all non-optional {microshift-short} RPM packages using the same version compatible with the `microshift-release-info` RPM.
+<2> References for automatically enabling {microshift-short} on system startup and applying default networking settings.
+<3> References for all non-optional {microshift-short} container images necessary for an offline MicroShift mode of operation.
+
 . Add the blueprint to the Image Builder by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints push minimal-microshift.toml
+$ sudo composer-cli blueprints push <minimal-microshift>.toml <1>
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:93]:
```suggestion
$ sudo composer-cli blueprints push microshift-blueprint.toml <1>
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 5]
```diff
@@ -39,21 +47,62 @@ EOF
...
 ----
+<1> Replace <minimal-microshift> with the name of your TOML file as needed.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:95]:
```suggestion
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 6]
```diff
@@ -39,21 +47,62 @@ EOF
...
 
 .Verification
 
 . Verify the Image Builder configuration listing only {microshift-short} packages by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift | grep microshift
+$ sudo composer-cli blueprints depsolve <minimal-microshift> | grep microshift <1>
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:103]:
```suggestion
$ sudo composer-cli blueprints depsolve microshift-blueprint | grep microshift
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 7]
```diff
@@ -39,21 +47,62 @@ EOF
...
 ----
+<1> Replace <minimal-microshift> with the name of your TOML file as needed.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:105]:
```suggestion
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 8]
```diff
@@ -71,5 +120,6 @@ blueprint: minimal-microshift v0.0.1
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift
+$ sudo composer-cli blueprints depsolve <minimal-microshift> <1>
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:123]:
```suggestion
$ sudo composer-cli blueprints depsolve microshift-blueprint.toml <1>
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 9]
```diff
@@ -71,5 +120,6 @@ blueprint: minimal-microshift v0.0.1
...
 ----
+<1> Replace <minimal-microshift> with the name of your TOML file as needed.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:125]:
```suggestion
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 10]
```diff
@@ -39,27 +47,68 @@ EOF
...
+<3> References for all non-optional {microshift-short} container images necessary for an offline MicroShift mode of operation.
+
 . Add the blueprint to the Image Builder by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints push minimal-microshift.toml
+$ sudo composer-cli blueprints push microshift-blueprint.toml <1>
 ----
+<1> Replace <microshift-blueprint> with the name of your TOML file as needed.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:95]: Do we need this repeated? We have it on the top of the doc.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 11]
```diff
@@ -39,27 +47,68 @@ EOF
...
 .Verification
 
 . Verify the Image Builder configuration listing only {microshift-short} packages by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift | grep microshift
+$ sudo composer-cli blueprints depsolve microshift-blueprint | grep microshift <1>
 ----
+<1> Replace <microshift-blueprint> with the name of your TOML file as needed.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:105]: Do we need this repeated? We have it on the top of the doc.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 12]
```diff
@@ -71,5 +120,6 @@ blueprint: minimal-microshift v0.0.1
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift
+$ sudo composer-cli blueprints depsolve microshift-blueprint.toml <1>
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:123]:
```suggestion
$ sudo composer-cli blueprints depsolve microshift-blueprint <1>
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 13]
```diff
@@ -71,5 +120,6 @@ blueprint: minimal-microshift v0.0.1
...
 ----
+<1> Replace <microshift-blueprint> with the name of your TOML file as needed.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:125]: Do we need this repeated? We have it on the top of the doc.

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 10 above)
Comment (ShaunaDiaz) [reply modules/microshift-adding-service-to-blueprint.adoc:95]: @ggiguash Yes, that's docs style. We don't know if users are going to come straight to the procedure and not see the top note.

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 11 above)
Comment (ShaunaDiaz) [reply modules/microshift-adding-service-to-blueprint.adoc:105]: @ggiguash Yes, that's docs style. We don't know if users are going to come straight to the procedure and not see the top note.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 14]
```diff
@@ -39,27 +47,68 @@ EOF
 The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.15.0` to download the {microshift-short} 4.15.0 RPMs.
 ====
 
+. Optional. Use the blueprint installed in the `/usr/share/microshift/blueprint` directory that is specific to platform architecture. See the following example snippet for an explanation of the blueprint sections:
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:50]:
```suggestion
. Optional. Use the blueprint installed in the `/usr/share/microshift/blueprint` directory that is specific to your platform architecture. See the following example snippet for an explanation of the blueprint sections:
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 15]
```diff
@@ -39,27 +47,68 @@ EOF
...
+
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dbc65f1fba7d92b36cf7514cd130fe83a9bd211005ddb23a8dc479e0eea645fd"
+...
+…
+EOF
+----
+<1> References for all non-optional {microshift-short} RPM packages using the same version compatible with the `microshift-release-info` RPM.
+<2> References for automatically enabling {microshift-short} on system startup and applying default networking settings.
+<3> References for all non-optional {microshift-short} container images necessary for an offline MicroShift mode of operation.
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:87]:
```suggestion
<3> References for all non-optional {microshift-short} container images necessary for an offline deployment.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 16]
```diff
@@ -39,27 +47,68 @@ EOF
...
+
 . Add the blueprint to the Image Builder by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints push minimal-microshift.toml
+$ sudo composer-cli blueprints push microshift-blueprint.toml <1>
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:93]:
```suggestion
$ sudo composer-cli blueprints push <microshift-blueprint.toml> <1>
```
User-replaced values always have the carrots wherever they are.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 17]
```diff
@@ -39,27 +47,68 @@ EOF
...
+<1> Replace <microshift-blueprint> with the name of your TOML file as needed.
 
 .Verification
 
 . Verify the Image Builder configuration listing only {microshift-short} packages by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift | grep microshift
+$ sudo composer-cli blueprints depsolve microshift-blueprint | grep microshift <1>
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:103]:
```suggestion
$ sudo composer-cli blueprints depsolve <microshift-blueprint> | grep microshift <1>
```

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 10 above)
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:95]:
```suggestion
<1> Replace <microshift-blueprint.toml> with the name of your TOML file as needed.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 18]
```diff
@@ -71,5 +120,6 @@ blueprint: minimal-microshift v0.0.1
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift
+$ sudo composer-cli blueprints depsolve microshift-blueprint <1>
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:123]:
```suggestion
$ sudo composer-cli blueprints depsolve <microshift-blueprint> <1>
```

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 11 above)
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:105]:
```suggestion
<1> Replace <microshift-blueprint> with the name of your TOML file.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 19]
```diff
@@ -9,16 +9,24 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* Follow step 1 to create your own minimal blueprint file which results in a faster minimal {microshift-short} installation.
+* Follow Step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
++
+[IMPORTANT]
+====
+Replace <microshift-blueprint> in the following procedures with the name of the TOML file you are using.
+====
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:18]:
```suggestion

```

Comment (agantony) [reply modules/microshift-adding-service-to-blueprint.adoc:18]: We are already adding callout to highlight this.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 20]
```diff
@@ -9,16 +9,24 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* Follow step 1 to create your own minimal blueprint file which results in a faster minimal {microshift-short} installation.
+* Follow Step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:13]:
```suggestion
To achieve a faster minimal {microshift-short} installation, create your own minimal blueprint file. This results in a faster installation process. After you have your blueprint file, use it for the installation. It contains all the required RPM packages and container images and enables a faster start as the container references are accessed locally. With this approach, you can optimize the installation process and achieve faster results.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 21]
```diff
@@ -9,16 +9,24 @@
...
++
+[IMPORTANT]
+====
+Replace <microshift-blueprint> in the following procedures with the name of the TOML file you are using.
+====
+
 .Procedure
 
-. Use the following example to create your blueprint:
+. Use the following example to create your own `microshift-blueprint.toml` blueprint file:
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:22]:
```suggestion
. Create your own `microshift-blueprint.toml` blueprint file:
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 22]
```diff
@@ -9,16 +9,24 @@
...
 +
-.Image Builder blueprint example
+.Custom Image Builder blueprint example
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:24]:
```suggestion
.Example custom image builder blueprint file:
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 23]
```diff
@@ -39,27 +47,68 @@ EOF
 The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.15.0` to download the {microshift-short} 4.15.0 RPMs.
 ====
 
+. Optional. Use the blueprint installed in the `/usr/share/microshift/blueprint` directory that is specific to your platform architecture. See the following example snippet for an explanation of the blueprint sections:
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:50]:
```suggestion
. Optional: Use the blueprint installed in the `/usr/share/microshift/blueprint` directory that is specific to your platform architecture:
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 24]
```diff
@@ -39,27 +47,68 @@ EOF
...
++
+.Generated Image Builder blueprint example snippet
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:52]:
```suggestion
.Example image builder blueprint file
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 25]
```diff
@@ -39,27 +47,68 @@ EOF
...
+
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dbc65f1fba7d92b36cf7514cd130fe83a9bd211005ddb23a8dc479e0eea645fd"
+...
+…
+EOF
+----
+<1> References for all non-optional {microshift-short} RPM packages using the same version compatible with the `microshift-release-info` RPM.
+<2> References for automatically enabling {microshift-short} on system startup and applying default networking settings.
+<3> References for all non-optional {microshift-short} container images necessary for an offline deployment.
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:87]:
```suggestion
<1> References for all non-optional {microshift-short} RPM packages that use the same version compatible with the `microshift-release-info` RPM.
<2> References for the automatic activation of {microshift-short} on system startup and the application of default network settings.
<3> References for all non-optional {microshift-short} container images required for offline use.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 26]
```diff
@@ -39,27 +47,68 @@ EOF
...
+
 . Add the blueprint to the Image Builder by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints push minimal-microshift.toml
+$ sudo composer-cli blueprints push <microshift-blueprint.toml> <1>
 ----
+<1> Replace <microshift-blueprint.toml> with the name of your TOML file.
```
Comment (agantony) [inline modules/microshift-adding-service-to-blueprint.adoc:95]:
```suggestion
<1> Specify the name of your TOML file.
``

Comment (agantony) [reply modules/microshift-adding-service-to-blueprint.adoc:95]: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks:~:text=%5Bsource%2Cterminal%5D%0A%2D%2D%2D%2D%0A%24%20oc%20autoscale,percent%20of%20requested%20CPU.

Comment (agantony) [reply modules/microshift-adding-service-to-blueprint.adoc:95]: same suggestion for all instances.

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 27]
```diff
@@ -39,27 +47,68 @@ EOF
...
+$ sudo composer-cli blueprints depsolve <microshift-blueprint> | grep microshift <1>
 ----
+<1> Replace <microshift-blueprint> with the name of your TOML file.
 +
 .Example output
 +
 [source,terminal]
 ----
-blueprint: minimal-microshift v0.0.1
+blueprint: microshift-blueprint v0.0.1
```
Comment (dhensel-rh) [inline modules/microshift-adding-service-to-blueprint.adoc:111]: I know that the example is potentially outside the scope of the review, but shouldn't this reflect the version we are targetting? 4.13 seems out of place here.

Comment (ggiguash) [reply modules/microshift-adding-service-to-blueprint.adoc:111]: Right, we can replace `4.13.1` by `4.15.1` at least.
I'm not sure what to do with the dates and commits though, i.e. `202305250827.p0.g4105d3`.

Maybe, we should remove this output all together?

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 28]
```diff
@@ -39,27 +47,68 @@ EOF
...
+<1> Replace <microshift-blueprint.toml> with the name of your TOML file.
 
 .Verification
 
 . Verify the Image Builder configuration listing only {microshift-short} packages by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift | grep microshift
+$ sudo composer-cli blueprints depsolve <microshift-blueprint> | grep microshift <1>
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:103]: I see that you added a template to the command: `<microshift-blueprint>`.
So, the <1> comment is not correct. It should be the `name` definition inside the TOML file.

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 27 above)
Comment (aravipra) [reply modules/microshift-adding-service-to-blueprint.adoc:111]: So Should I remove this output file?

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 28 above)
Comment (aravipra) [reply modules/microshift-adding-service-to-blueprint.adoc:103]: I can replace the existing reference with the below
<1> Replace <microshift-blueprint> with the name tag in your TOML file.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 29]
```diff
@@ -71,5 +120,6 @@ blueprint: minimal-microshift v0.0.1
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift
+$ sudo composer-cli blueprints depsolve <microshift-blueprint> <1>
```
[the remark this answers is not in the document]
Comment (aravipra) [reply modules/microshift-adding-service-to-blueprint.adoc:123]: I can replace the existing reference with the below
<1> Replace <microshift-blueprint> with the name tag in your TOML file.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 30]
```diff
@@ -9,16 +9,25 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* Follow step 1 to create your own minimal blueprint file which results in a faster minimal {microshift-short} installation.
+* Follow Step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:13]:
```suggestion
* Follow step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 31]
```diff
@@ -9,16 +9,25 @@
...
+. Use the following example to create your own blueprint file:
 +
-.Image Builder blueprint example
+.Custom Image Builder blueprint example
 +
 [source,text]
 ----
-cat > minimal-microshift.toml <<EOF
-name = "minimal-microshift"
+cat > microshift_blueprint.toml <<EOF <1>
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:29]:
```suggestion
cat > _<microshift_blueprint.toml>_ <<EOF <1>
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 32]
```diff
@@ -9,16 +9,25 @@
...
+name = "microshift_blueprint" <2>
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:30]:
```suggestion
name = "_<microshift_blueprint>_" <2>
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 33]
```diff
@@ -9,16 +9,25 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* Follow step 1 to create your own minimal blueprint file which results in a faster minimal {microshift-short} installation.
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:12]:
```suggestion
* Follow step 1 to create your own minimal blueprint file which results in a faster {microshift-short} installation.
```

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 31 above)
Comment (aravipra) [reply modules/microshift-adding-service-to-blueprint.adoc:29]: This is an actual image builder blueprint example. so can this have <>?

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 32 above)
Comment (aravipra) [reply modules/microshift-adding-service-to-blueprint.adoc:30]: same comment as above

Comment (ShaunaDiaz) [reply modules/microshift-adding-service-to-blueprint.adoc:30]: same reply
